README.md

```diff
@@ -104,6 +104,7 @@ try {
   console.log(`Block confirmations = ${txStats.confirmations}`)
   console.log(`Transaction fee = ${txStats.feeSatoshi}`)
   console.log(`Transaction size = ${txStats.sizeBytes}`)
+  console.log(`Transaction size = ${txStats.dataHex}`)
```

Comment:
The console log statement is incorrect here.

Reply:
Hi, it's for a project I am working on, similar to Blockcerts: https://github.com/blockchain-certificates
I need to read the hash stored in a transaction with OP_RETURN.

Reply:
Corrected.